src/main/java/racingcar/dto/PlayerDto.java

```diff
@@ -0,0 +1,22 @@
+package racingcar.dto;
```

Comment:
해당 dto 는 어느 패키지에 위치하는것이 좋을까요?

Reply:
PlayerDto는 DB에서 받은 값이기에 repository 디렉터리 안으로 dto 패키지를 이동했습니다 !

```suggestion
package racingcar.repository.dto;
```

<img width="284" alt="image" src="https://user-images.githubusercontent.com/82203978/233419956-1330b6ee-3947-4064-abd3-aad263209ae5.png">